azure-pipelines.yml

```diff
@@ -1,5 +1,8 @@
-# https://docs.microsoft.com/vsts/pipelines/languages/dotnet-core
-
+# https://docs.microsoft.com/en-us/azure/devops/pipelines/yaml-schema
+trigger:
+  - +:refs/heads/*
```

Comment:
If this isn't working for you, please try: `  - refs/heads/*` (remove the `+:`).

Reply:
Odd, I set up a build myself w/ those settings and tags builds are working for me.  Could you check a couple of things for me:
1) Do you have a PR trigger override in your pipeline definition? (Edit pipeline->Triggers Tab->PR trigger.  Is the checkbox enabled? ...mine is clear.)
2) How are you creating the tag?  (FWIW, I tested using the GitHub releases UI.)

Reply:
Thanks @lkillgore, I'll check this out on Monday